lib/qxcli/commands/Create.js

```diff
@@ -166,7 +167,7 @@ qx.Class.define("qxcli.commands.Create", {
           choices: v.choices,
           name: var_name,
           message,
-          default : deflt,
+          default : v.default,
```

Comment:
This needs to be "deflt"  - otherwise it breaks in case `v.default` is a function. Please revert your change here.

Reply:
No, default can be a function: https://www.npmjs.com/package/inquirer

default: (String|Number|Array|Function) Default value(s) to use if nothing is entered, or a function that returns the default value(s). If defined as a function, the first parameter will be the current inquirer session answers.

Reply:
Ok, didn't know that. Thank you. Then all the "deflt" stuff can go away.

Reply:
Not completely. In line 163 you need the calculated value.